src/Components/PresentationalComponents/CVEPageDetailsDescription/CVEPageDetailsExpandable.js

```diff
@@ -0,0 +1,53 @@
+import { Text, TextContent } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+
+const CVEPageDetailsExpandable = props => {
+    const [isDescriptionTruncated, toggleDescriptionTruncated] = useState(true);
+    const [isDescriptionTooLong, toggleDesciptionTooLong] = useState(false);
+    const [truncatedDescription, setTruncatedDescription] = useState('');
+
+    const { characterLimit, showMoreLabel, showLessLabel, content } = props;
+
+    useEffect(() => {
+        if (String(content).length > characterLimit) {
+            toggleDesciptionTooLong(true);
+            setTruncatedDescription(props.content.slice(0, characterLimit - 1) + '…');
+        }
+        else {
+            toggleDesciptionTooLong(false);
+        }
+    }, []);
+
+    const toggleDescription = () => {
+        toggleDescriptionTruncated(!isDescriptionTruncated);
+    };
+
+    if (isDescriptionTooLong) {
+        return (
+            <TextContent>
+                <Text component={props.component}>{isDescriptionTruncated ? truncatedDescription : content}</Text>
+                <a rel="noopener noreferrer" onClick={toggleDescription}>
+                    {isDescriptionTruncated ? showMoreLabel || 'Show more' : showLessLabel || 'Show less'}
```

Comment:
you can define default values for your props by assigning to `defaultProps ` 

an example: https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values